<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>1300</width>
     <height>990</height>
    </size>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tamanho da Caixa</string>
    </property>
   </widget>
   <widget class="Plotter" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>531</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>171</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_5"/>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="sliderCz">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_6"/>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_7"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="sliderCx">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sliderCy">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>171</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QSlider" name="sliderRaio">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_4"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Raio da Esfera</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>171</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="0">
      <widget class="QSlider" name="SliderA">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Opacidade</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cor do Voxel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="sliderG">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Verde</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Vermelho</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Azul</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sliderR">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="sliderB">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSair"/>
    <addaction name="actionSalvar_2"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEscultor"/>
   <addaction name="actionPutVoxel"/>
   <addaction name="actionCutVoxel"/>
   <addaction name="actionPutBox"/>
   <addaction name="actionCutBox"/>
   <addaction name="actionPutSphere"/>
   <addaction name="actionCutSphere"/>
   <addaction name="actionPutEllipsoid"/>
   <addaction name="actionCutEllipsoid"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEscultor">
   <property name="text">
    <string>Escultor</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionPutVoxel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PutVoxel</string>
   </property>
  </action>
  <action name="actionCutVoxel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CutVoxel</string>
   </property>
  </action>
  <action name="actionPutBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PutBox</string>
   </property>
  </action>
  <action name="actionCutBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CutBox</string>
   </property>
  </action>
  <action name="actionPutSphere">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PutSphere</string>
   </property>
  </action>
  <action name="actionCutSphere">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CutSphere</string>
   </property>
  </action>
  <action name="actionPutEllipsoid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PutEllipsoid</string>
   </property>
  </action>
  <action name="actionCutEllipsoid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CutEllipsoid</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>OFF</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionSalvar_2">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header location="global">plotter.h</header>
   <container>1</container>
   <slots>
    <slot>alteraCor()</slot>
    <slot>mudaB(int)</slot>
    <slot>mudaG(int)</slot>
    <slot>salvaEscultor()</slot>
    <slot>executaGeomview()</slot>
    <slot>limpaEscultor()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionPutVoxel</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>execTipo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCutVoxel</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>execTipo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPutBox</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>execTipo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCutBox</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>execTipo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPutSphere</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>execTipo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCutSphere</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>execTipo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPutEllipsoid</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>execTipo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCutEllipsoid</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>execTipo(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSair</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Sair()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSalvar_2</sender>
   <signal>triggered()</signal>
   <receiver>widget</receiver>
   <slot>salvaEscultor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCx</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>814</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>839</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCy</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>804</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>830</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCz</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>842</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRaio</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>826</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>845</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>capturaAcao(bool)</slot>
  <slot>Sair()</slot>
  <slot>Salvar()</slot>
  <slot>execTipo(bool)</slot>
 </slots>
</ui>
